File: C++/Basic_Problem-cpp/TriangleArea.cpp
// author: Sazzad
#include <bits/stdc++.h>
using namespace std;

int main()
{
  double a, b, c ,s , area ;

  cout<< "Enter the firts value : " ;
  cin>> a ;
  cout<< "Enter the second value : " ;
  cin>> b ;
  cout<< "Enter the third value : " ;
  cin>> c ; 
  if ((a + b)> c && (b + c)>a && (a + c)> b )
  {
    s = (a + b + c)/2 ;
    area = sqrt(s * (s - a) * (s - b) * (s - c)) ;
    cout<< "The triangle area is : " << area <<endl ;
  }
  else
  {
    cout<< "Triangle is not possible ." <<endl ;
  }
  return 0;
}

File: Python/Fundamental/triangleArea.py
import math
def triangle():
    a = int(input("Enter the value of A:"))
    b = int(input("Enter the value of B:"))
    c = int(input("Enter the value of C:"))
    if ((a+b)>c and (b+c)>a and (a+c)>b):
        s = (a+b+c)/2
        area = math.sqrt(s*(s-a)*(s-b)*(s-c))
        print("The triangle area is:",area)
    else:
        print("Triangle is not possible.")
triangle()



"Using Class         :    -------------"
import math
class trianglearea:
    def __init__(self,a,b,c):
        if ((a+b)>c and (b+c)>a and (a+c)>b):
            s = (a+b+c)/2
            area = math.sqrt(s*(s-a)*(s-b)*(s-c))
            print("Triagnle are is : ",area)
        else:
            print("Triangle is not possible")
a = int(input("Enter the value of A:"))
b = int(input("Enter the value of B:"))
c = int(input("Enter the value of C:"))
tarea = trianglearea(a,b,c)